Firebase/android/app/src/main/res/layout/merge_login_view.xml

```diff
@@ -8,18 +8,19 @@
     android:layout_height="match_parent"
     android:orientation="vertical">
 
-    <FrameLayout
+    <com.novoda.bonfire.view.BubblyLinearLayout
```

Comment:
The other place where it is used has multiple views inside the layout so extended the layout that would work in both places